### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by bitflame** [ImageMagick/ImageMagick#3276] [General]
Greetings-
Is this an active project, and is there an opportunity for me to learn and contribute? I looked, and did not see any reference about it.

**Comment by snibgo** [comment]
ImageMagick is widely used. I suppose it is installed on millions, maybe billions, of computers. The software constantly evolves. So, yes, I suppose it is "an active project".

You can learn about it by reading documentation. Above all, learn by using it.

You can contribute by answering questions on this forum, or by suggesting changes to the software, or contributing money.

**Comment by dlemstra** [comment]
We also have a couple up for grabs issues: https://github.com/ImageMagick/ImageMagick/labels/up%20for%20grabs

**Comment by bitflame** [comment]
Thanks.

**Reply by dlemstra** [reply]
Feel free to reach out here or by email if you need some help.

**Comment by bitflame** [comment]
Thank you so much. I will. I am so glad I found out about this project.
